roles/solr_collection/README.md

```diff
@@ -0,0 +1,38 @@
+Role Name
+=========
+
+A brief description of the role goes here.
```

Comment:
There are so many autogenerated files in this new setup, we'll probably need to investing some time into deciding how detailed we want to make these files / how we want to structure tests, documentation, and linting going forward

Reply:
I agree — what would be a good way / when would we be at a good point to go through the PUL example together and revise it to match our current best practices and approach?